src/pages/student_detail/StudentDetail.tsx

```diff
@@ -36,81 +85,189 @@ const StudentDetail = () => {
-          </InformationContainer>
-          <InformationContainer>
-            <Label>Día de pago:</Label>
-            <StudentInfo>
-              {student.paymentDay ||
-                (student.planType == "Principio de mes" && "Del 1 al 10")}
-            </StudentInfo>
-          </InformationContainer>
-        </div>
-      </StudentInfoContainer>
-      <PaymentsContainer>
-        <SubTitle>PAGOS</SubTitle>
-        <PaymentsTable>
-          <TableHeader>
-            <tr>
-              <HeaderProperty>Pago N°</HeaderProperty>
-              <HeaderProperty>Fecha</HeaderProperty>
-              <HeaderProperty>Monto</HeaderProperty>
-              <HeaderProperty>Estado</HeaderProperty>
-              <HeaderProperty>Pagado</HeaderProperty>
-            </tr>
-          </TableHeader>
-          <TableBody>
-            {student?.payments?.map((payment) => (
-              <Row key={payment.number}>
-                <PaymentInfo>{payment.number}</PaymentInfo>
-                <PaymentInfo>{payment.paymentDate}</PaymentInfo>
-                <PaymentInfo>{formatCurrency(payment.amount)}</PaymentInfo>
-                <PaymentInfo
-                  color={payment.status === "Vencido" ? "red" : "black"}
-                >
-                  {payment.status}
-                </PaymentInfo>
-                <PaymentInfo>
-                  {payment.paymentDate && <img src={CheckIcon} width={30} />}
-                </PaymentInfo>
-              </Row>
-            ))}
-          </TableBody>
-        </PaymentsTable>
-      </PaymentsContainer>
+      <HeaderContainer>
+        <Title>
+          <img
+            src={BackIcon}
+            alt="back-icon"
+            onClick={() => navigate(-1)}
+            style={{ cursor: "pointer" }}
+          ></img>
+          DETALLE DEL ALUMNO
+        </Title>
+        <ButtonWrapper>
+          <Button
+            size="medium"
+            variant={isActive ? "warning" : "success"}
+            icon={
+              isActive ? (
+                <img src={DesactivateIcon} alt="desactivate-icon" />
+              ) : (
+                <img
+                  src={DesactivateIcon}
+                  alt="activate-icon"
+                  style={{ transform: "rotate(180deg)" }}
+                />
+              )
+            }
+            onClick={handleOpenConfirm}
+          >
+            {isActive ? "Dar de baja" : "Dar de alta"}
+          </Button>
+          {showConfirm && (
+            <ConfirmDialog
+              message={`¿Estás seguro de ${
+                isActive ? "dar de baja" : "dar de alta"
+              } a ${student.name} ${student.lastName}?`}
+              onConfirm={handleConfirm}
+              onCancel={handleCancel}
+            />
+          )}
+        </ButtonWrapper>
+      </HeaderContainer>
+
+      <StudentNameContainer>
+        <Title>
+          <IconStyles>
+            <img src={StudentIcon} alt="student-icon"></img>
```

Comment:
Aca podes directamente cerrarlo asi


```suggestion
            <img src={StudentIcon} alt="student-icon" />
```